rocky/rocky/urls.py

```diff
@@ -84,7 +84,7 @@
         name="organization_add",
     ),
     path(
-        "<organization_code>/<path:pk>/edit/",
+        "<organization_code>/<int:pk>/edit/",
```

Comment:
The pk is not necessary here actually, we already have the code, so this can just be `<organization_code>/edit/`

Reply:
Edited in https://github.com/minvws/nl-kat-coordination/pull/318/commits/1602747e044eec605a21b070131e1a33448541b6